.readthedocs.yaml

```diff
@@ -0,0 +1,12 @@
+version: 2
+build:
+  os: ubuntu-22.04
+  tools:
+    nodejs: "20"
+  commands:
+    - npm install
+    - npm run build
+    - npm run dev
```

Comment:
I want to avoid `npm run dev` just for the sake of principles, but running `node packages/mystmd/dist/myst.cjk` did not seem to work. I thought there was a field in `package.json` that is equivalent to the `pyprojet.toml:project.scripts`. Having something like that would be useful to be more straightforward on how to run with the build folder artifacts.